include/controller.hpp

```diff
@@ -86,6 +88,8 @@ the RC.
             int &motorFR,
             int &motorBL,
             int &motorBR,
+            int &motorML,
```

Comment:
`motorML` and `motorMR` are not (or, at least, should not be) defined for quadcopters